packages/yoastseo/src/scoring/assessments/inclusiveLanguage/configuration/ageAssessments.js

```diff
@@ -5,7 +5,17 @@ import { includesConsecutiveWords } from "../helpers/includesConsecutiveWords";
 import { SCORES } from "./scores";
 import notInclusiveWhenStandalone from "../helpers/notInclusiveWhenStandalone";
 
+/*
```

Comment:
I'd suggest the change the order of these lines: first he what the part of the code does (e.g. "The last sentence of strings for the non-inclusive terms for 'older people/person'." and then after the quoted string). seems more logical. feel free to disregard though, the rest looks great!

Reply:
I also thought that it's a more logical order, but also if you're hovering over the variable to see what it says, it's easier to see that if it's at the beginning of the comment. Compare the first and the second screenshots. Or would you suggest to change it despite that?

<img width="439" alt="Screenshot 2023-01-19 at 14 39 41" src="https://user-images.githubusercontent.com/32479012/213457742-ba05157a-869e-4006-aa91-516986a5c72f.png">
<img width="415" alt="Screenshot 2023-01-19 at 14 40 12" src="https://user-images.githubusercontent.com/32479012/213457781-d168cc00-53cc-4b7a-ad17-e5794a0e7588.png">

Reply:
Hm I actually would suggest despite that, because the dev may be looking both for what the variable is/does (for example, is it a feedback string or a sentence from a feedback string?), _and_ the feedback string it refers to.